core/base/inc/TROOT.h

```diff
@@ -314,6 +314,7 @@ friend TROOT *ROOT::Internal::GetROOT2();
    void              Reset(Option_t *option="");
    void              SaveContext();
    void              SetApplication(TApplication *app) { fApplication = app; }
+   /// Set the "Batch mode". "True" means the session is without graphics
```

Comment:
As far as I know the session won't be "without graphics" (you can still create canvases, paint things on them, save them to file). Maybe "without interactive graphics"? I'm not sure, you are the expert :smile:

Reply:
Indeed.  As hinted by the spelling, this is intended to force the non-interactive (for the GUI part) mode (i.e. batch mode as in running on batch systems).